src-tauri/src/util.rs

```diff
@@ -143,3 +143,39 @@ pub fn check_northstar_running() -> bool {
         || s.processes_by_name("Titanfall2.exe").next().is_some();
     x
 }
+
+#[allow(dead_code)]
+pub fn copy_dir_all(
```

Comment:
same family of functions as `move_dir_all` generally useful and will become quickly relevant for #444